pkg/controller/podautoscaler/replica_calculator_test.go

```diff
@@ -197,7 +202,7 @@ func (tc *replicaCalcTestCase) prepareTestClient(t *testing.T) (*fake.Clientset,
 			return true, nil, fmt.Errorf("no custom metrics specified in test client")
 		}
 
-		assert.Equal(t, tc.metric.name, getForAction.GetMetricName(), "the metric requested should have matched the one specified")
+		assert.Equal(t, (tc.metric).name, getForAction.GetMetricName(), "the metric requested should have matched the one specified")
```

Comment:
what's up with the extra parens?  those aren't necessary

Reply:
Removed